jobs/consul-controller-dns-setup/spec

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
don't you need an empty "monit" file for this job as well?

Reply:
jobs/consul-controller-dns-setup/monit is present in the pull request.